### DISCUSSION TITLE
Now playing on the home screen to short..?

### DISCUSSION THREAD

**Discussion by kaptainen** [music-assistant/support#1930] [Feature requests and ideas]
![Screenshot_20240219_164854_Fully Kiosk Browser](https://github.com/music-assistant/hass-music-assistant/assets/30773649/b3852e4f-5e67-406e-8d28-b49861004ac6)

Any chance off having the now playing on the home screen full width of the music assistant,  or dynamic adjusting automatically to full width.. know my dashboard is a little special with the iframe showing the music assistant, but it's the same in the Companion app.. hope it makes sense.

**Comment by OzGav** [comment]
They are cards. If you have more than one player playing then they appear next to each other and are eventually scrollable.

**Reply by kaptainen** [reply]
Ahh, thanks for the reply.. Just tried playing on a few players at once and see it now...
